Tests/SwiftParserTest/translated/UnclosedStringInterpolationTests.swift

```diff
@@ -26,99 +26,90 @@ final class UnclosedStringInterpolationTests: XCTestCase {
   func testUnclosedStringInterpolation2() {
     AssertParse(
       ##"""
-      _ = 1️⃣"mid == \(pete"
+      _ = "mid == \(pete1️⃣"2️⃣
       """##,
       diagnostics: [
-        // TODO: Old parser expected error on line 1: cannot find ')' to match opening '(' in string interpolation
-        // TODO: Old parser expected error on line 1: unterminated string literal
-        DiagnosticSpec(message: "expected expression"),
-        DiagnosticSpec(message: #"extraneous code '"mid == \(pete"' at top level"#),
+        DiagnosticSpec(locationMarker: "1️⃣", message: #"unexpected code '"' in string literal"#),
+        DiagnosticSpec(locationMarker: "2️⃣", message: "expected ')' in string literal"),
+        DiagnosticSpec(locationMarker: "2️⃣", message: #"expected '"' to end string literal"#),
       ]
     )
   }
 
   func testUnclosedStringInterpolation3() {
     AssertParse(
       ##"""
-      let theGoat = 1️⃣"kanye \("
+      let theGoat = "kanye \("1️⃣
       """##,
       diagnostics: [
-        // TODO: Old parser expected error on line 1: cannot find ')' to match opening '(' in string interpolation
-        // TODO: Old parser expected error on line 1: unterminated string literal
-        DiagnosticSpec(message: "expected expression in variable"),
-        DiagnosticSpec(message: #"extraneous code '"kanye \("' at top level"#),
+        DiagnosticSpec(message: #"expected '"' to end string literal"#),
+        DiagnosticSpec(message: "expected ')' in string literal"),
+        DiagnosticSpec(message: #"expected '"' to end string literal"#),
```

Comment:
Same as 2, though honestly I'm not sure it's worth keeping both. That's true for most of the `testUncloseStringInterpolation*` TBH.